### PR TITLE
Focusing Greenfield Installation on Cloud Shell usage

### PR DESCRIPTION
**Pull request by draychev**
This PR augments the Greenfield installation documentation of AGIC so that it is usable with [Azure Cloud Shell](https://shell.azure.com/)
It also provides the simplest possible app that we could install on an AKS in one bash command.

### CODE REVIEW

File: docs/setup/install-new.md [hunk 1]
```diff
@@ -67,116 +67,158 @@ Example: ![Deployment Output](../images/deployment-output.png)
...
-1. **Application Gateway Ingress Controller** - This is the controller which monitors ingress-related events and actively keeps your Azure Application Gateway installation in sync with the changes within the AKS cluster.
 
-Steps:
 
-1. Configure `kubectl` with access to your newly deployed AKS:
+### Install App Gateway Ingress Controller
+Ingress Controller will monitor ingress-related events and will keep Azure Application Gateway
+in sync with the changes within the AKS cluster.
+
+1. Add [aad-pod-identity](https://github.com/Azure/aad-pod-identity) service to your cluster:
```
Comment (akshaysngupta) [inline docs/setup/install-new.md]: Can you move add AAD Pod Identity service to the previous section ?

Comment (draychev) [reply docs/setup/install-new.md]: I created `Prerequisites` section and in it now we have `Setup Kubernetes Credentials` and also `Install AAD Pod Identity` -- is that what you meant?

Also simplified the sample app YAML (but it won't work until we apply this fix: https://github.com/Azure/application-gateway-kubernetes-ingress/pull/553)

Comment (akshaysngupta) [reply docs/setup/install-new.md]: I mean can you move it to #### Install AAD Pod Identity

Comment (draychev) [reply docs/setup/install-new.md]: Got it! Makes sense!

**Comment by draychev** [comment]
@akshaysngupta PTAL

**Pull request merged**